wasm/src/lib.rs

```diff
@@ -196,6 +199,28 @@ extern "C" {
     pub fn log(s: &str);
 }
 
+#[macro_export]
+macro_rules! array {
+        ($($value:expr),*$(,)?) => {{
+            let array = Array::new();
```

Comment:
I know this isn't your fault, this code was just copy-pasted from earlier code, but for extra robustness it should use `::js_sys::Array::new()` and `::wasm_bindgen::JsValue::from($value)`